docs/tools/vdb_table/data/aperturedb.json

```diff
@@ -67,93 +67,93 @@
     "comment": ""
   },
   "embeddings_text": {
-    "support": "",
-    "source_url": "",
-    "comment": ""
+    "support": "full",
+    "source_url": "https://docs.aperturedata.io/HowToGuides/Applications/semantic_search/website_ingest",
+    "comment": "We use text embeddings in ApertureDB vector store for our documentation chatbot."
   },
   "embeddings_image": {
-    "support": "",
-    "source_url": "",
-    "comment": ""
+    "support": "full",
+    "source_url": "https://docs.aperturedata.io/HowToGuides/Advanced/Image%20Similarity%20Search",
+    "comment": "Supports multimodal embeddings in general including for videos."
   },
   "embeddings_structured": {
-    "support": "",
-    "source_url": "",
-    "comment": ""
+    "support": "partial",
+    "source_url": "https://docs.aperturedata.io/query_language/Reference/descriptor_commands/desc_commands/AddDescriptor",
+    "comment": "ApertureDB vector store can store embeddings from any source but we don't generate them ourselves."
   },
   "rag": {
-    "support": "",
-    "source_url": "",
-    "comment": ""
+    "support": "full",
+    "source_url": "https://docs.aperturedata.io/category/chatbot-with-langchain-and-aperturedb",
+    "comment": "You can build hybrid-RAG with our graph-vector search."
   },
   "recsys": {
-    "support": "",
-    "source_url": "",
-    "comment": ""
+    "support": "partial",
+    "source_url": "https://docs.aperturedata.io/HowToGuides/Applications/similarity_search",
```

Comment:
"Built-in support for recommendation system functionality, e.g. a solution that refines the results based on behavioral data."

Reply:
I removed this but to me - recommendation logic is a layer above any DB. DB provides the data for the algorithms.